shared/src/main/scala/scala/xml/Utility.scala

```diff
@@ -105,7 +104,7 @@ object Utility extends AnyRef with parsing.TokenTests {
     val escMap = pairs map { case (s, c) => c -> ("&%s;" format s) }
     val unescMap = pairs ++ Map("apos" -> '\'')
   }
-  import Escapes.{ escMap, unescMap }
+  import Escapes.unescMap
```

Comment:
I believe the claim that `escMap` was unused was a bit apocryphal on the compiler's part.

Can the compiler determine if an import is unused by other source files as well?